docs/guides/05-data-driven-visualizations-part-2.md

```diff
@@ -0,0 +1,176 @@
+## Data-driven visualizations (part 2)
+
+In the previous section we talked about using different types of inputs for ramp, but we always output colors picked from a list. `ramp` supports to use other types of outputs and also CARTO VL includes some fixed constant palettes of colors. Let's see it!
+
+### Color values
+
+One way to output colors is to specify a list of colors, just like we have done in all the previous examples. This can be done with expressions like `ramp($dn, [blue, red])`. In those examples, you mainly saw the usage of named colors (`blue`, `red`, `green`, etc.). With CARTO VL any valid color expression (not just named colors) can be used:
+
+- `ramp($dn, [rgb(200,220,222), rgba(200,120,22, 0.8)])`
+- `ramp($dn, [hsv(0,1,1), hsv(0.5,1,1)]`
+- `ramp($dn, [#00F, #F00])`
+- `ramp($dn, [blue, #F00])`
+- `ramp($dn, [opacity(blue, 0.4), opacity( #F00, 0.6),])`
+
+Another way to specify colors is to use built-in color schemes. CARTO VL, by default, supports the use of both [CARTOColors](https://carto.com/carto-colors/) and [ColorBrewer](http://colorbrewer2.org) schemes.
+
+To access CARTOColors, replace the list of colors with the name of the scheme you want to use:
+
+- `ramp($dn, temps)`
+- `ramp($dn, purpor)`
+
+To access ColorBrewer, replace the list of colors with the name of the scheme you want to use appended with `cb_`:
+
+- `ramp($dn, cb_blues)`
+
+You can find scheme names for CARTOColors [here](https://carto.com/carto-colors/) and for ColorBrewer [here](http://colorbrewer2.org).
+
+Use the map below, to switch between various color expressions and color schemes:
+
+```CARTO_VL_Viz
+// Style 1: apply colors from a color list of named colors
+
+color: ramp($dn, [green, yellow, red])
```

Comment:
The original styles showed some things these examples don't: ability to mix different forms of colors (like named colors with `rgb()`), hex colors and `opacity`

Reply:
@davidmanzanares I didn't actually remove any of what you had... there were the series of examples that were in the text (and are kept like that):
```
- `ramp($dn, [rgb(200,220,222), rgba(200,120,22, 0.8)])`
- `ramp($dn, [hsv(0,1,1), hsv(0.5,1,1)]`
- `ramp($dn, [#00F, #F00])`
- `ramp($dn, [blue, #F00])`
- `ramp($dn, [opacity(blue, 0.4), opacity( #F00, 0.6),])`
```
from that section, I see this one: `ramp($dn, [opacity(blue, 0.4), opacity( #F00, 0.6),])`

and for the map, i didn't add, remove, or modify any of the styles i just extracted the radio button text to the code block above... the map. style 2 uses tranparent + named colors

```
// Style 1: apply colors from a color list of named colors

color: ramp($dn, [green, yellow, red])

// Style 2: apply colors from a color list including "transparent"

color: ramp($dn, [transparent, gray, yellow])

// Style 3: apply colors from a color list with HSV color constructors

color: ramp($dn, [hsv(0,1,1), hsv(0.5,1,1)])

// Style 4: apply color using the CARTOColor scheme "temps"

color: ramp($dn, temps)
```
Maybe I accidentally got rid of an example (?) anyway, feel free to add back... wasn't intentional.

Reply:
Ah, sorry I was trying to compare this to the previous version and I compared the styles here with the styles given a few lines before (which I didn't see in this version, but, as you point out, they are there), Sorry for the noise, everything is good :slightly_smiling_face: